content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -587,7 +587,17 @@ For MongoDB in versions `3.9.0` and later, Prisma Client exposes three methods t
 
 ### <inlinecode>$runCommandRaw</inlinecode>
 
-`$runCommandRaw` runs a raw MongoDB command against the database. For example, this query inserts two records with the same `_id`, bypassing normal document validation:
+`$runCommandRaw` runs a raw MongoDB command against the database. As input, it accepts all [MongoDB database commands](https://www.mongodb.com/docs/manual/reference/command/), with the following exceptions:
+
+- find (use [`aggregateRaw`](#aggregateraw) instead)
+- aggregate (use [`findRaw`](#findraw) instead)
+
+When you use `$runCommandRaw` to run a MongoDB database command, note the following:
+
+- The object that you pass when you invoke `$runCommandRaw` must follow the syntax of the MongoDB database command.
```

Comment:
Style guide: should we have a rule to start a bullet point with a lowercase letter when following a stem sentence?

Reply:
I've already got this topic in the style guide questions list. However, in this case, it's a separate sentence - the lead-in sentence is complete.